src/neptune/common/deprecation.py

```diff
@@ -25,13 +29,22 @@ class NeptuneDeprecationWarning(DeprecationWarning):
 warnings.simplefilter("always", category=NeptuneDeprecationWarning)
 
 warned_once = set()
+path_to_root_module = os.path.dirname(os.path.realpath(neptune.__file__))
+
+
+def get_user_code_stack_level():
```

Comment:
Is it finally working with `run[path] =` and `run[path].function(args)`?

If so, congrats!

Reply:
Everything ❤️ 
Just look at these messages:
![Screenshot 2023-01-17 at 14 26 48](https://user-images.githubusercontent.com/917619/212910951-6a82ff3a-0cd0-44be-80af-5dbb94eccd50.png)